README.md

```diff
@@ -17,6 +17,30 @@ You can find an FAQ with answers to issues at [FAQ](https://github.com/slimgroup
 
 You can find installation instruction in our Wiki at [Installation](https://github.com/slimgroup/JUDI.jl/wiki/Installation)
 
+## GPU
+
+JUDI supports the computation of the wave equation on GPU via [Devito](https://www.devitoproject.org)'s GPU offloading support.
+
+**NOTE**: Only the wave equation part will be computed on GPU, the julia arrays will still be CPU arrays and `CUDA.jl` is not supported.
+
+### Installation
+
+To enable gpu support in JUDI, you will need to install one of [Devito](https://www.devitoproject.org)'s supported offloading compilers. We strongly recommend checking the [Wiki](https://github.com/devitocodes/devito/wiki) for installation steps and to reach out to the Devito community for GPU compiler related issues.
```

Comment:
This somehow contradicts the fact that Devito could be installed automatically if JUDI can't find it.

Reply:
There is a massive difference bettween installing Devito, that is the python package, and the the compiler (gcc, icc, ....) that ae used to compile the generated C code. And while JUDI will install Devito, it is up to the user to install proper compiler that are system and hardware dependent is up to the user.